package.json

```
@@ -38,25 +38,25 @@
     "babel-loader": "^8.2.4",
     "babel-preset-env": "^1.7.0",
     "circular-dependency-plugin": "^5.2.2",
-    "css-loader": "^5.2.7",
-    "html-loader": "^1.3.2",
-    "html-webpack-plugin": "^4.5.2",
+    "css-loader": "6.7.3",
```

Patches and minor flexible version should be safe enough (the same is for all other packages as well)

```suggestion
    "css-loader": "^6.7.3",
```